beard/utils/names.py

```diff
@@ -46,3 +50,69 @@ def normalize_name(name):
 def name_initials(name):
     """Compute the set of initials of a given name."""
     return set([w[0] for w in name.split()])
+
+RE_APOSTROPHES = re.compile('\'+')
+RE_REMOVE_NON_CHARACTERS = re.compile('[^a-zA-Z\',\s]+')
+DROPPED_PREFIXES = {'a', 'ab', 'am', 'ap', 'abu', 'al', 'auf', 'aus', 'bar',
+                    'bath', 'bat', 'ben', 'bet', 'bin', 'bint', 'd', 'da',
+                    'dall', 'dalla', 'das', 'de', 'degli', 'del', 'dell',
+                    'della', 'dem', 'den', 'der', 'di', 'do', 'dos', 'du', 'e',
+                    'el', 'i', 'ibn', 'im', 'jr', 'l', 'la', 'las', 'le',
+                    'los', 'm', 'mac', 'mc', 'mhic', 'mic', 'o', 'ter', 'und',
+                    'v', 'van', 'vom', 'von', 'zu', 'zum', 'zur'}
+
+
+@memoize
+def tokenize_name(name):
```

Comment:
I think we should be more explicit in the function name. Something like `dm_tokenize_name` maybe.